v3/util/time.go

```diff
@@ -60,8 +60,10 @@ var (
 	MozillaPolicy24Date         = time.Date(2017, time.February, 28, 0, 0, 0, 0, time.UTC)
 	MozillaPolicy241Date        = time.Date(2017, time.March, 31, 0, 0, 0, 0, time.UTC)
 	MozillaPolicy27Date         = time.Date(2020, time.January, 1, 0, 0, 0, 0, time.UTC)
+	BALLOT_SC_12_Ineffective    = time.Date(2019, time.April, 1, 0, 0, 0, 0, time.UTC)
```

Comment:
```suggestion
	BallotSC12Ineffective    = time.Date(2019, time.April, 1, 0, 0, 0, 0, time.UTC)
```

WDYT? (e.g. lines 47-62)

Reply:
Thanks, I hop back-and-forth between Go/Rust/Python/Java often enough that the naming conventions bleed over each other sometimes.